DependencyInjection/DoctrineFixturesExtension.php

```diff
@@ -29,5 +31,8 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(dirname(__DIR__).'/Resources/config'));
 
         $loader->load('services.xml');
+
+        $container->registerForAutoconfiguration(ORMFixtureInterface::class)
```

Comment:
Lovely - we shouold copy this for the MongoDBODMBundle as well.

Reply:
Yes! Exactly :)